Problem2.py

```diff
@@ -0,0 +1,38 @@
+#Time Complexity - O(n)
+#Space Complexity - O(n) "Worst Case"
+#Made a HashMap with id as key and value as employee object
```

Comment:
Good Job! Diksha. Please try to explain what n stands for in time and space complexity . Everything else looks good.